packages/example-log-extension/README.md

```diff
@@ -0,0 +1,364 @@
+# loopback4-example-log-extension
```

Comment:
Should we change this to `example-log-extension`?

Reply:
I'll use `@loopback/example-log-extension` for consistency with getting-started example (I'll fix package.json too).